testing/scripts/seldon_e2e_utils.py

```diff
@@ -42,6 +43,27 @@ def wait_for_shutdown(deploymentName, namespace):
         ret = run(f"kubectl get -n {namespace} deploy/{deploymentName}", shell=True)
 
 
+def get_deployment_names(seldonDeploymentName, namespace, attempts=20, sleep=5):
+    for _ in range(attempts):
+        ret = run(
+            f"kubectl get -n {namespace} seldondeployment {seldonDeploymentName} -o yaml",
+            shell=True,
+            capture_output=True,
+        )
+        if ret.returncode == 0:
+            logging.warning(
+                f"Successfully waited for SeldonDeployment {seldonDeploymentName}"
+            )
+            break
+        logging.warning(
+            f"Unsuccessful wait command but retrying for SeldonDeployment {seldonDeploymentName}"
+        )
+        time.sleep(sleep)
+    assert ret.returncode == 0
+    data = yaml.safe_load(ret.stdout.decode())
+    return list(data["status"]["deploymentStatus"])
```

Comment:
So deploymentStatus is actually deployment name? Interesting, maybe would be worth adding a small comment as it's not as intuitive

Reply:
It is actually a dictionary, casting to `list` returns keys. I will add appropriate comment.

Reply:
Added appropriate comment.